attestation-exploit/blank.yml

```
@@ -0,0 +1,92 @@
+    name: Attestation Exploit 
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)